src/main/java/build/buildfarm/worker/CASFileCache.java

```diff
@@ -214,7 +234,7 @@ private void expireDirectory(Digest digest) throws IOException {
         fileEntry.containingDirectories.remove(digest);
       }
     }
-    Worker.removeDirectory(path);
+    CASFileCache.removeDirectory(path);
```

Comment:
this is the same file, just do removeDirectory(path).

Reply:
I could have sworn I saw a linting warning here, but maybe that's only on non-self-class invocations.